.github/workflows/proselint.yml

```diff
@@ -0,0 +1,42 @@
+name: proselint
+
+on:
+  push:
+    paths:
+    - book/**
```

Comment:
@chuawjk would you like this workflow to be applicable to all branches including `main`? or just on feature branches?

Reply:
I think it's ok to have it apply to all branches. Is there a significant downside to having it run on main as well?

Reply:
No downside. Just highlighting a redundancy. Otherwise all good.